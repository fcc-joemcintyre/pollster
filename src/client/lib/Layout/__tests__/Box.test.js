/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Box } from '..';

test ('Box renders default component', () => {
  const component = renderer.create (
    <Box />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Box renders centered component', () => {
  const component = renderer.create (
    <Box center />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Box renders color component', () => {
  const component = renderer.create (
    <Box bg='blue' />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Box renders inline component', () => {
  const component = renderer.create (
    <Box inline />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
