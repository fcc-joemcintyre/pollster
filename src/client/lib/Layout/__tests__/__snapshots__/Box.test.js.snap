// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders centered component 1`] = `
.c0 {
  margin-left: auto;
  margin-right: auto;
}

<div
  className="c0"
/>
`;

exports[`Box renders color component 1`] = `
.c0 {
  background-color: blue;
}

<div
  className="c0"
/>
`;

exports[`Box renders default component 1`] = `
<div
  className=""
/>
`;

exports[`Box renders inline component 1`] = `
.c0 {
  display: inline-block;
}

<div
  className="c0"
/>
`;
